sdk/containerregistry/Azure.Containers.ContainerRegistry/tests/ContainerRegistryBlobClientLiveTests.cs

```diff
@@ -0,0 +1,279 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Containers.ContainerRegistry.Specialized;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Containers.ContainerRegistry.Tests
+{
+    [NonParallelizable]
+    public class ContainerRegistryBlobClientLiveTests : ContainerRegistryRecordedTestBase
```

Comment:
Is it worth adding tests to validate the error experience?

Reply:
I will add more tests in a subsequent PR.